src/template.html

```diff
@@ -1,55 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<title><%= htmlWebpackPlugin.options.title %></title>
-		<meta name="viewport" content="width=device-width, initial-scale=1.0, minimal-ui">
-		<meta name="color-scheme" content="dark light" />
-		<meta name="mobile-web-app-capable" content="yes">
-		<meta name="apple-mobile-web-app-capable" content="yes">
-		<meta name="apple-touch-icon" content="/assets/app-icon.png" sizes="512x512">
-		<link rel="manifest" href="<%= htmlWebpackPlugin.files.publicPath %>manifest.json">
-		<meta name="description" content="Preact is a fast 3kB alternative to React with the same modern API.">
-		<% if (htmlWebpackPlugin.options.manifest.theme_color) { %>
-			<meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>">
-		<% } %>
-		<link rel="alternate" type="application/rss+xml" href="https://preactjs.com/feed.xml" />
-		<link rel="alternate" type="application/atom+xml" href="https://preactjs.com/feed.atom" />
-		<meta name="twitter:card" content="summary" />
-		<meta property="og:url" content="https://preactjs.com<%= htmlWebpackPlugin.options.url %>" />
-		<meta property="og:title" content="<%= htmlWebpackPlugin.options.title %>" />
-		<meta property="og:description" content="<%= htmlWebpackPlugin.options.CLI_DATA.preRenderData.description %>" />
-		<meta property="og:image" content="https://preactjs.com/assets/app-icon.png" />
-		<% if (process.env.BRANCH) { %>
-			<script>ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga('set','dimension1','<%= process.env.BRANCH %>');onerror=function(e,f,l,c){ga('send','event','exception',e,f+':'+l+':'+c)}</script>
-		<% } %>
-		<% const loadManifest = htmlWebpackPlugin.options.createLoadManifest(compilation.assets, webpack.namedChunkGroups);%>
-		<% const filesRegexp = /\.(css|js)$/;%>
-		<% for (const file in loadManifest[htmlWebpackPlugin.options.url]) { %>
-			<% if (htmlWebpackPlugin.options.preload && file && file.match(filesRegexp)) { %>
-				<% /* crossorigin for main bundle as that is loaded from `<script type=module` tag, other lazy loaded bundles are from webpack so its not needed */ %>
-				<link rel="preload" href="<%= htmlWebpackPlugin.files.publicPath + file %>" as="<%= file.match(/\.css$/)?'style':'script' %>"<%= file.match(/bundle\.\w{5}\.esm\.js$/)?' crossorigin="anonymous"':'' %>>
-			<% } %>
-		<% } %>
```

Comment:
Preloading was removed from `preact-cli`, as flawed beyond redemption really.

It worked by assuming directories/files in `src/routes` matched the URL for that resource, so `src/routes/profile` -> `/profile`.

As such, it'd fall apart if users tried to map `src/routes/notFound` -> `/404` or, in our case here, we didn't use the `routes/` directory. This whole block never actually resulted in preload links because of that.